CHANGELOG.md

```diff
@@ -0,0 +1,55 @@
+# Change log
```

Comment:
Has this been adjusted to fit only changes made on `firehose-aptos`. I see `sfnear` mention. Some of the entries are also not relevant because they were not changed. Please adjust it.